packages/wallets/src/features/cfd/screens/MT5TradeScreen/MT5TradeLink/MT5TradeLink.tsx

```diff
@@ -15,25 +15,36 @@ type TMT5TradeLinkProps = {
 };
 
 const MT5TradeLink: FC<TMT5TradeLinkProps> = ({ app = 'linux', isDemo = false, platform, webtraderUrl = '' }) => {
-    const { data: ctraderToken } = useCtraderServiceToken();
+    const { refetch } = useCtraderServiceToken();
```

Comment:
if I understand correctly - the flow of this change will result in token request being send twice: on render, and then again on click.
It should be called once - you can use either: `    const { send } = useAPI();` or useMutation()
(I recommend just using send - useMutation in this case is just a wrapper over send() from useApi, so kinda pointless)

Reply:
I think we should only call it once, when its needed, we should avoid duplicated calls especially for non-static data. I would say its worth implementing the change, especially given mt5 is quite popular (meaning potentially a number of duplicated calls)

Reply:
Sure